### PR TITLE
Remove ownCloud migration

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

> *"Si en 2021 on n'a pas fait la migration, on a raté sa vie"*
>  Jacques S.

## Solution
- *Remove ownCloud migration script*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [ ] **Code review** : 
- [ ] **Approval (LGTM)** : 
- [ ] **Approval (LGTM)** : 
- **CI succeeded** : 
[Build Status](https://ci-apps-hq.yunohost.org/jenkins/job/nextcloud_ynh%20PR-NUM-/)  
*Please replace '-NUM-' in this link by the PR number.*  
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

**Review by kay0u — APPROVED** [review]
Ok, you got me!

File: scripts/upgrade [hunk 1]
```diff
@@ -87,44 +87,11 @@ then
 
     # Remove the option backup_core_only after the backup.
     ynh_app_setting_delete $app backup_core_only
-
-    ynh_clean_setup () {
```
Comment (kay0u) [inline scripts/upgrade:91]: Uuuh, in fact, why did you removed the ynh_clean_setup function?

Comment (ericgaspar) [reply scripts/upgrade:91]: true, it's a mistake

**Review by kay0u — APPROVED** [review]
:+1:

**Pull request merged**